include/hip/hip_runtime_api.h

```diff
@@ -114,6 +114,7 @@ typedef struct hipDeviceProp_t {
     int isMultiGpuBoard;                      ///< 1 if device is on a multi-GPU board, 0 if not.
     int canMapHostMemory;                     ///< Check whether HIP can map host memory
     int gcnArch;                              ///< AMD GCN Arch Value. Eg: 803, 701
```

Comment:
Please update the comment and all README that this is returning only major numbering only.

Reply:
Updated the comment and addressed other review comments.